docs/install/install-from-packages.md

```diff
@@ -38,6 +36,17 @@ You may host multiple Plone sites on the same server.
 -   Minimum 512 MB hard disk space is required.
     40 GB or more hard disk space is recommended.
 
+````{warning}
+When you _build_ Plone, it will approximately double RAM requirements to _install_ Plone.
```

Comment:
Huh? Building and installing are the same, no? The distinction we really want to make is that installing Plone takes more RAM than running a basic Plone site, right?

Reply:
In this context of a cookiecutter install, I thought we install already built Docker images. Is that not correct?

Reply:
@stevepiercy No, not really.

Running the cookiecutter creates a project structure including a Makefile with various commands. It does not immediately run any of those commands.

Some of the commands install and run Plone without using containers: `make install`, `make start-backend`, `make start-frontend`.

Some of the commands build Docker images for the project (based on the released base images, plus project-specific code) and run Plone using them: `make build-images`, the various commands starting with `stack-`, and the various commands related to acceptance tests.

All of these require the RAM to build Plone (particularly I think it's the webpack compilation that is RAM-intensive)

Reply:
OK, then maybe this?

```
When you _install_ or _build_ Plone, it will approximately double RAM requirements to _run_ Plone.
```

Also is the preceding bullet point even true anymore?

```
-   Minimum 256 MB RAM and 512 MB of swap space per Plone site is required.
    2 GB or more RAM per Plone site is recommended.
```

Reply:
How about:

Installing and running the Plone backend requires at least 256 MB of RAM and 512 MB of swap space per Plone site.
2 GB or more RAM per Plone site is recommended.
Building the Volto frontend requires 2 GB of RAM.

Reply:
Wasn't there something about lxml when installing Plone backend that bumped up RAM requirements? Ah, maybe swap? https://community.plone.org/t/struggling-to-install-plone-on-ubuntu-droplet/3301/6

Reply:
@stevepiercy hmm, ok, I didn't know about that. Although these days in most cases on Linux pip will be able to fetch a prebuilt binary wheel for lxml rather than building it in situ.

How about something along these lines:

Running Plone requires at least 512 MB of RAM per Plone site. 2 GB or more RAM per Plone site is recommended.
2 GB or more RAM may also be needed temporarily while installing Plone and building the frontend assets with webpack. On a machine without much RAM, one way to get through the installation is to add swap.

Reply:
This is a messy section. See new PR: https://github.com/plone/documentation/pull/1561